apps/common-app/src/apps/reanimated/examples/CopyShareablesPerformanceTest.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import { Button, Keyboard,StyleSheet, Text, TextInput, TouchableWithoutFeedback, View } from 'react-native';
```

Comment:
Looks like code is unformatted
```suggestion
import { Button, Keyboard, StyleSheet, Text, TextInput, TouchableWithoutFeedback, View } from 'react-native';
```